doc/src/manual/methods.md

```diff
@@ -263,10 +263,11 @@ Possible fix, define
   g(::Float64, ::Float64)
 ```
 
-Here the call `g(2.0, 3.0)` could be handled by either the `g(Float64, Any)` or the `g(Any, Float64)`
-method, and neither is more specific than the other. In such cases, Julia raises a [`MethodError`](@ref)
-rather than arbitrarily picking a method. You can avoid method ambiguities by specifying an appropriate
-method for the intersection case:
+The order in which the methods are defined does not matter and neither is more specific
+than the other. Here the call `g(2.0, 3.0)` could be handled by either the
+`g(Float64, Any)` or the `g(Any, Float64)` method. In such cases, Julia raises a
```

Comment:
```suggestion
Here the call `g(2.0, 3.0)` could be handled by either the `g(::Float64, ::Any)` or the
`g(::Any, ::Float64)` method. The order in which the methods are defined does not matter and
neither is more specific than the other. In such cases, Julia raises a
```
I would put the second sentence first. Saying `neither is more specific` before even naming the methods in question seems confusing.

The suggestion also changes `g(Float64, Any)` to `g(::Float64, ::Any)`.